src/services/MpdfQrProvider.php

```diff
@@ -27,9 +30,7 @@ class MpdfQrProvider implements IQRCodeProvider
      * @param array<int> $foreground RGB foreground and border color. Default [0, 0, 0].
      * @param int $compression Compression level: from 0 (default, no compression) to 9.
      */
-    public function __construct(public array $background = array(255, 255, 255), public array $foreground = array(0, 0, 0), public int $compression = 0)
-    {
-    }
+    public function __construct(public array $background = array(255, 255, 255), public array $foreground = array(0, 0, 0), public int $compression = 0) {}
```

Comment:
coding standard says:

> The soft limit on line length MUST be 120 characters.

> Lines SHOULD NOT be longer than 80 characters; lines longer than that SHOULD be split into multiple subsequent lines of no more than 80 characters each.

regardless, I agree that multiline is better and improves readability. for another time though, I really want to focus on bugs rather than linting for now.